mycroft/util/log.py

```diff
@@ -84,7 +84,10 @@ def init(cls):
         # Check configs manually, the Mycroft configuration system can't be
         # used since it uses the LOG system and would cause horrible cyclic
         # dependencies.
-        confs = [SYSTEM_CONFIG, USER_CONFIG]
+        confs = []
+        for dir in BaseDirectory.load_config_paths('mycroft'):
+            confs.append(join(dir, 'mycroft.conf'))
+        confs.append('/etc/mycroft/mycroft.conf')
```

Comment:
Shouldn't this one be the very first config loaded? this way it takes precedence over the user config, everywhere else in core it doesn't

Reply:
But the user config should always takes precedence over the system config. It doesn't make sense the other way around.
This way you can configure something system-wide but have different users have separate options.

Reply:
It gets reversed later, I assume it had to do with how the XDG is ordered.

Reply:
Jarbas seems to be right, the merge below will apply the /etc/... last making it take precedence

Reply:
Oh he is saying it currently takes precedence over the user config? That would be wrong indeed then. @forslund since it's now your PR, could you make this change? :sweat_smile:

Reply:
_Note: Github won't let me create a new thread about the same line of code. I can see that this now gets reversed on the following line._

Separate to that, should this be appending `SYSTEM_CONFIG` rather than a hard coded string?

Reply:
It should yes